paddle/fluid/platform/stream_callback_manager.h

```diff
@@ -18,8 +18,8 @@
 #include <cuda_runtime.h>
 #include <functional>
 #include <memory>
-#include "ThreadPool.h"
 #include "paddle/fluid/platform/enforce.h"
+#include "third_party/threadpool/src/extern_threadpool/ThreadPool.h"
```

Comment:
Please clean up this line. If the include path does not take effect, we need to fix the include cmakelist

Reply:
ok